packages/cf-deploy-config-writer/templates/router/package.json

```diff
@@ -1,18 +1,10 @@
 {
     "name": "app-router",
-    "private": true,
```

Comment:
Was this always incorrect or why the change?

Reply:
The approuter configurations have always been a replication of the route folder generated by cds and in the latest version, this property is now removed. Here are some notes that I've taken from different forums;
Cleaner Build Pipelines:
Some SAP Cloud SDK and CAP deployment setups involve using tools like npm pack, npm ci, or custom scripts to bundle and deploy services (including approuter). Having "private": true would block packaging steps in these cases.

CDS Tooling Evolution:
SAP’s Yeoman-based generators and cds add approuter have evolved toward being more flexible and universal. Not enforcing private: true reduces friction and lets developers decide whether they want that flag based on their own packaging and deployment needs.